pym/gentoolkit/eclean/cli.py

```diff
@@ -478,7 +479,7 @@ def optionSwitch(option, opts, action=None):
     options["verbose"] = False
     options["changed-deps"] = False
     options["ignore-failure"] = False
-    options["clean-invalid"] = False
+    options["clean-invalid"] = True
```

Comment:
Should this instead be `no-clean-invalid` being False? Not sure, please check very carefully if the logic works correctly.

Reply:
If its False then it will be off by default no ?

Reply:
I just wanted to make sure that there wasn't confusion between `clean-invalid` and `no-clean-invalid` as options, if you get what I mean.

Reply:
(Really, we should replace the whole arg parsing with `argparse` or maybe `click`, but that's for another day.)

Reply:
Ah you mean consistency okay okay I'll make it consistent

Reply:
Better now ?